File: app/src/main/java/com/druger/aboutwork/adapters/MyReviewAdapter.java
package com.druger.aboutwork.adapters;

import android.view.ViewGroup;

import androidx.recyclerview.widget.RecyclerView;

import com.druger.aboutwork.model.Review;

import java.util.List;

/**
 * Created by druger on 14.12.2017.
 */

public class MyReviewAdapter extends ReviewAdapter {

    public MyReviewAdapter(List<Review> reviews) {
        super(reviews);
    }

    @Override
    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
        return super.onCreateViewHolder(parent, viewType);
    }

    @Override
    public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
        super.onBindViewHolder(holder, position);
    }
}

File: app/src/main/java/com/druger/aboutwork/adapters/MyReviewAdapter.kt
package com.druger.aboutwork.adapters

import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView

/**
 * Created by druger on 14.12.2017.
 */

class MyReviewAdapter : ReviewAdapter() {

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
        return super.onCreateViewHolder(parent, viewType)
    }

    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
        super.onBindViewHolder(holder, position)
    }
}
